basic/payment-splitter/script.sh

```diff
@@ -0,0 +1,20 @@
+resim reset
+
+export op1=$(resim new-account)
```

Comment:
I am getting `error: Found argument 'account' which wasn't expected, or isn't valid in this context` when running this script

Reply:
Apologies, `script.sh` was a file that I was experimenting with internally and did not mean to commit it. I've removed this file in the most recent commit as I did have similar issues with it as well.